batavia/modules/dis.js

```diff
@@ -84,6 +84,7 @@ batavia.modules.dis = {
         batavia.modules.dis.def_op('DUP_TOP_TWO', 5);
 
         batavia.modules.dis.def_op('NOP', 9);
+        batavia.modules.dis.NOP = 9;
```

Comment:
Why are both the assignment and `def_op` needed? This is probably more for my education than an actual concern.